ButterCMS/ButterCMSClient.cs

```diff
@@ -53,6 +56,7 @@ public ButterCMSClient(string authToken, TimeSpan? timeOut = null, int maxReques
                 Timeout = timeOut ?? defaultTimeout,
                 BaseAddress = new Uri(apiBaseAddress)
             };
+            httpClient.DefaultRequestHeaders.Add("X-Butter-Client", ".NET/" + Assembly.GetExecutingAssembly().GetName().Version.ToString());
```

Comment:
Will this call always work? `Assembly.GetExecutingAssembly().GetName().Version.ToString())`

Looks like a lot of things chained together. Other SDK's look at the string value of the package (i.e. https://github.com/ButterCMS/buttercms-csharp/pull/6/files#diff-7144306aafc08bb01102d83d01328464R5)  and import + string concat that.

I have no idea which is better for .NET but just wanted to mention that. The set of chained calls looks a little scary.